### ISSUE TITLE
ChordPro Error: Fatal Problems found

### ISSUE THREAD

**Issue by JohannesWiesner** [ChordPro/chordpro#203]
## Describe the bug
 I installed ChordPro version using the Windows installer. When inserting the example and clicking on `Preview` I get the following error:
 
 ```
Fatal Problems found!
No such file or directory
Fatal problems found!
 ```

### Expected behavior
Adobe Pdf reader should open with the rendered file

### Attachments

![fatal_error_chordpro](https://user-images.githubusercontent.com/48254290/150760974-2da61e87-48f0-45e7-8fab-e8d1d245c288.png)

## System information
Windows 10

**Comment by sciurius** [comment]
Please expand the Details so we can see what is going on.
My guess is that you didn't configure a PDF viewer in the preferences.

**Comment by JohannesWiesner** [comment]
Hi, if I click on the details, I don't get more information than what I've already posted ('Fatal Error' / 'No such file or director'). Strange enough, now the error already happens when I click on Preview..But you're right, I didn't specify a PDF viewer (though I remember, that I used ChordPro already a couple of months ago and I am pretty sure that I never specified a Reader). The field is empty. Should I point to an .exe for a PDF viewer? Does \ / matter when defining the filepath?
